fuzz/Makefile

```diff
@@ -20,10 +20,10 @@ PROTO_IN = $(ROOT_DIR)/fuzz/proto
 PROTO_OUT = $(ROOT_DIR)/fuzz/proto/gen
 
 ifneq ($(FUZZ_ENV), ossfuzz)
-CC = clang++
```

Comment:
wish we could have cmake support too.

Reply:
I am going to create a task/issue for this.